### PR TITLE
feat(voting_escrow): ability to display vxASTRO balance and logo

### PR DESCRIPTION
**Pull request by epanchee**

### CODE REVIEW

File: contracts/voting_escrow/src/contract.rs [hunk 1]
```diff
@@ -76,6 +83,49 @@ pub fn instantiate(
     )?;
     BLACKLIST.save(deps.storage, &vec![])?;
 
+    let mint = match msg.mint {
+        Some(m) => Some(MinterData {
+            minter: addr_validate_to_lower(deps.api, &m.minter)?,
```
Comment (wedancedalot) [inline contracts/voting_escrow/src/contract.rs:88]: minter is always env.contract.address

Comment (epanchee) [reply contracts/voting_escrow/src/contract.rs]: Ok, fixed

File: contracts/voting_escrow/src/contract.rs [hunk 2]
```diff
@@ -565,6 +624,10 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
                 deposit_token_addr: config.deposit_token_addr.to_string(),
             })
         }
+        QueryMsg::Balance { address } => to_binary(&get_user_balance(deps, env, address)?),
+        QueryMsg::TokenInfo {} => to_binary(&query_token_info(deps)?),
```
Comment (maksk7) [inline contracts/voting_escrow/src/contract.rs]: total_supply in TokenInfo's response is always 0. It's better to return total voting power as a total supply in response.

Comment (epanchee) [reply contracts/voting_escrow/src/contract.rs]: Yep, thanks. Already fixed

**Review by ShadoySV — APPROVED** [review]

**Pull request merged**